core/include/vmx.h

```diff
@@ -681,7 +681,7 @@ void vmx_vmwrite(struct vcpu_t *vcpu, const char *name,
             vmwrite(vcpu, GUEST_##seg##_AR, tmp_ar);               \
         }
 
-#elif defined(__MACH__)
+#elif defined(HAX_PLATFORM_DARWIN)
```

Comment:
I guess this is probably to workaround a compiler issue (MSVC vs. Clang), because the Mac code differs from the Windows code only in an extra pair of parentheses. But since we've finished testing the PR, you can address this in a later patch when you add Linux support.